app/views/sessions/purchase_cart.html.erb

```diff
@@ -7,7 +7,11 @@
     </tr>
     <% if session[:cart] %>
         <% session[:cart].each do |item_id, quantity| %>
-        <% item = Item.find_by(id: item_id) %>
+            <% item = Item.find_by(id: item_id) %>
+            <!-- Line below generates receipt for purchasing this item, if possible -->
+            <% if logged_in? %>
```

Comment:
That seems a lot less intuitive to a user to me though. Primarily because the tool can be a guided process, while the URL requires the user to edit something that most individuals likely don't seriously edit.